backend/views/core/settings/view.py

```diff
@@ -2,6 +2,8 @@
 from django.http import HttpRequest
 from django.shortcuts import render
 from django.contrib.auth import update_session_auth_hash
+from PIL import Image
```

Comment:
I think we should swap this for
from django.core.validators import validate_image_file_extension